missilewars-plugin/src/main/java/de/butzlabben/missilewars/game/equipment/PlayerEquipmentRandomizer.java

```diff
@@ -110,18 +112,32 @@ private void sendRandomGameEquipment() {
         int randomID;
 
         // switch between type of "items":
-        // after 2 missile items, you get one special item
+        // after 2 missile items, you get one special item or a shield
         if (sendEquipmentCounter >= 2) {
 
-            randomID = randomizer.nextInt(equipmentManager.getSpecialEquipment().getSpecialEquipmentList().size());
-            item = equipmentManager.getSpecialEquipment().getSpecialEquipmentList().get(randomID);
+            // Special Equipment or Schematic Game-Equipment of the type "Shield":
+            
+            int specialEquipment = equipmentManager.getSpecialEquipment().getSpecialEquipmentList().size();
+            int shieldsEquipment = equipmentManager.getShieldEquipment().getSchematicEquipmentList().size();
+            
+            randomID = randomizer.nextInt(1, specialEquipment + shieldsEquipment + 1);
+            if (randomID <= specialEquipment) {
+                item = equipmentManager.getSpecialEquipment().getSpecialEquipmentList().get(randomID - 1);
```

Comment:
I fixed several logic bugs in this method: The randomizer works from 1 (included) to X (excluded). And the `EquipmentList.get()` argument need the `randomID - 1` to prevent some out-of-bound-exceptions.